third_party/VulkanTutorial/OffscreenRenderingVulkanTutorial.cpp

```diff
@@ -230,22 +230,13 @@ void OffscreenRenderingVulkanTutorial::CreateOffscreenImage() {
   };
   image_extent_ = create_info.extent;
 
-  for (int format_int = VK_FORMAT_BEGIN_RANGE; format_int <= VK_FORMAT_END_RANGE; ++format_int) {
-    const auto format = static_cast<VkFormat>(format_int);
-
-    VkImageFormatProperties format_properties{};
-    VkResult result = vkGetPhysicalDeviceImageFormatProperties(
-        physical_device_, format, create_info.imageType, create_info.tiling, create_info.usage,
-        create_info.flags, &format_properties);
-    if (result == VK_SUCCESS) {
-      image_format_ = format;
-      break;
-    }
-  }
-
-  ORBIT_CHECK(image_format_ != VkFormat::VK_FORMAT_UNDEFINED);
+  VkImageFormatProperties format_properties{};
+  VkResult result = vkGetPhysicalDeviceImageFormatProperties(
```

Comment:
So we are choosing a `VkFormat` that we know to work instead of looking for one. That sounds good, I don't remember why it wasn't done in the first place. Two things though:

- `image_format_` is in a group of member variables that are meant to be initialized in the initialization phase of the tutorial, but is used as a constant now. It can be removed, or better turned into a constant next to `static constexpr uint32_t kImageWidth = 800; static constexpr uint32_t kImageHeight = 600;`.
- This call to `vkGetPhysicalDeviceImageFormatProperties` looks like it can be removed: before it was used to find out which format can be used, but now we are probably just fine with the `CHECK_VK_SUCCESS(vkCreateImage(device_, &create_info, nullptr, &image_));` that comes later?

Reply:
Regarding the first: Yes, that sounds good. Changed!

Regarding the second: I removed the call and vkCreateImage still succeeds. So it's probably fine.

PTAL!